src/app/utils/dataLoaderUtil.js

```diff
@@ -53,6 +49,9 @@ const successCb = (pathType, dispatch) => (response) => {
 
 function loadDataForRoutes(location, dispatch) {
   const { pathname } = location;
+  if (pathname === `${baseUrl}/`) {
```

Comment:
Is this going to cause any issues with auth? I wonder if we have any problems coming from patron data being reset if they hit the home page after logging in on another page.

Reply:
The reset state action currently preserves the patron and `appConfig`.